### DISCUSSION TITLE
개인정보 수집을 고려해 보면 좋을 것 같아요.

### DISCUSSION THREAD

**Discussion by zangsu** [woowacourse-teams/2024-code-zap#614] [결정 사항]
런칭 페스티벌부터 다른 사람의 글을 볼 수 있는 기능이 추가되었습니다. 
이제는 "나의 코드를 잘 관리한다" 라는 것만으로 우리 서비스를 설명하기 어려워졌는데요. 

본인이 작성한 글이 불특정 다수에게 공개되는 순간, 악의적인 사용자를 고려하는 것이 좋을 것 같은데요. 

우리가 어렵게 유치한 사용자를 잃고 싶지는 않기 때문에, 미리 악성 사용자를 차단할 준비를 해 두는 것도 좋을 것 같습니다.

+) 개인정보 수집 대신 악의적인 게시글을 작성하지 못하도록 특정 키워드를 포함하는 게시글 업로드를 막아 줄 수도 있을 것 같아요.

**Comment by zangsu** [comment]
법적으로 주민번호 수집을 금지하고 있기 때문에 이메일을 수집해야 할 것 같아요.
제안드리는 사항은 다음과 같습니다.
1. 빠른 시일 내에 이메일을 수집한다. 
    1. 이메일 검증이 불가능하다면 이메일 검증은 미룬다.
2. 최종적으로 이메일 검증을 추가한다.

**Comment by zeus6768** [comment]
사용자 정보 수집의 법적 책임에 대비하기보다는 우회할 방법을 찾는게 좋다고 생각해요. 

짱수, 켬미, 몰리, 초롱은 OAuth 2.0 도입을 어떻게 생각하시나요?

**Reply by zangsu** [reply]
전 문제만 해결할 수 있으면 OAuth 도입도 환영입니다!!

**Reply by HoeSeong123** [reply]
찬성입니다. 짱수가 얘기한 문제도 해결할 수 있을 것 같고, 좋은 도전과제가 될 것 같아요.
